Library/Homebrew/cli/args.rb

```diff
@@ -5,13 +5,14 @@
 module Homebrew
   module CLI
     class Args < OpenStruct
-      attr_accessor :processed_options
+      attr_accessor :processed_options, :args_parsed
```

Comment:
```suggestion
      attr_reader :processed_options, :args_parsed
```

Reply:
https://github.com/Homebrew/brew/pull/6433/files#diff-e0e03670c451d934ae1d79adfff63cacR144
cannot be executed with only `attr_reader`

Reply:
Ok but I think it would be better to have a custom method that allows them to only be set once and then be frozen/not allow future changes to be made rather than using `attr_accessor`

Reply:
@MikeMcQuaid What do you think about this approach?